package.json

```diff
@@ -16,7 +16,8 @@
     "typesync": "turbo run typesync",
     "knip": "knip",
     "validate-config": "turbo run validate-config",
-    "merge-checks": "turbo run publint manypkg typesync knip typecheck lint validate-config",
+    "merge-checks": "turbo run publint manypkg typesync knip typecheck lint validate-config check-deduped-deps",
+    "check-deduped-deps": "pnpm dedupe --check",
```

Comment:
what's the point of deduping?

Reply:
If you've got `foo@1.0` which depends on `baz@1.0`, and then `baz` gets a new release, then install `bar@1.0` which depends on `baz@1.1`, then you'll get two `baz`es, as it won't upgrade the other baz. Then, PNPM will end up downloading and install many different copies over time, which is both wasteful and can break types.